department-of-veterans-affairs/va.gov-cms
# VACMS-2572: Add patch and empty checks to prevent empty array notices for tablefield.

## Description
See #2572 

## Testing done
Drupal log review

## Screenshots
N/A

## QA steps

- Open drupal logs, and filter by php errors. Take note of time of most recent entry for `Notice: Undefined index: caption in Drupal\tablefield\Plugin\Field\FieldFormatter\TablefieldFormatter->viewElements() (line 145 of /app/docroot/modules/contrib/tablefield/src/Plugin/Field/FieldFormatter/TablefieldFormatter.php)`
- Go to `/va-oklahoma-city-health-care/locations/ada-va-clinic` and refresh page
- Go back to log page, filter by php, and confirm there aren't any newer instances of `Notice: Undefined index: caption in Drupal\tablefield\Plugin\Field\FieldFormatter\TablefieldFormatter->viewElements() (line 145 of /app/docroot/modules/contrib/tablefield/src/Plugin/Field/FieldFormatter/TablefieldFormatter.php)`
